Tiltfile

```diff
@@ -0,0 +1,166 @@
+# -*- mode: Python -*-
+
+update_settings(k8s_upsert_timeout_secs=60)  # on first tilt up, often can take longer than 30 seconds
+
+# set defaults
+settings = {
+    "allowed_contexts": [
+        "kind-bmo"
+    ],
+    "preload_images_for_kind": True,
+    "kind_cluster_name": "bmo",
+    "enable_providers": [],
+}
+
+keys = []
+
+always_enable_providers = ["metal3-bmo"]
+providers = {}
+extra_args = settings.get("extra_args", {})
+
+# global settings
+settings.update(read_json(
+    "tilt-settings.json",
+    default = {},
+))
+
+if settings.get("trigger_mode") == "manual":
+    trigger_mode(TRIGGER_MODE_MANUAL)
+
+if "allowed_contexts" in settings:
+    allow_k8s_contexts(settings.get("allowed_contexts"))
+
+if "default_registry" in settings:
+    default_registry(settings.get("default_registry"))
+
+def load_provider_tiltfiles(provider_repos):
+    for repo in provider_repos:
+        file = repo + "/tilt-provider.json"
+        provider_details = read_json(file, default = {})
+        if type(provider_details) != type([]):
+            provider_details = [provider_details]
+        for item in provider_details:
+            provider_name = item["name"]
+            provider_config = item["config"]
+            if "context" in provider_config:
+                provider_config["context"] = repo + "/" + provider_config["context"]
+            else:
+                provider_config["context"] = repo
+            if "kustomize_config" not in provider_config:
+                provider_config["kustomize_config"] = True
```

Comment:
Having a bit difficulty understanding this if `kustomize_config` is not in `provider config` so setting it to true will do what?